Documentation/UpdatingToGA.md

```diff
@@ -2,8 +2,36 @@
 
 Between the RC2 and GA releases of the Microsoft Mixed Reality Toolkit, changes were made that may impact existing projects. This document describes those changes and how to update projects to the GA release.
 
+- [API changes](#api-changes)
 - [Assembly name changes](#assembly-name-changes)
 
+## API changes
+
+Since the release of RC2, there have been a number of API changes including some that may break existing projects. The following sections describe the changes that have occurred between the RC2 and GA releases.
+
+- [Breaking changes](#breaking-changes)
```

Comment:
I think that by definition, all of these changes in this document are breaking changes. It's not just the API stuff that's breaking.

I would just delete the "breaking changes" line and header down below, as it seems to suggest the others AREN'T breaking (not true)